sched/job_def_serializer_test.go

```diff
@@ -1,3 +1,5 @@
+// +build property
```

Comment:
reading https://jira.twitter.biz/browse/SCOOT-145, it seems there's some desire to be able to run unit tests without having to wait for property tests.